src/main/java/com/home/onlineshop/service/impl/DBServices/WareCategoryServiceImpl.java

```diff
@@ -33,7 +32,17 @@ public WareCategoryDto create(String wareCategoryName) {
     }
 
     @Override
-    public Iterable<WareCategory> getAll() {
-        return wareCategoryRepository.findAll();
+    public Iterable<WareCategoryDto> getAll() {
+        return wareCategoryMapper.wareCategoryToDto(wareCategoryRepository.findAll());
+    }
+
+    @Override
+    @Transactional
+    public boolean delete(Long id) {
+        if(wareCategoryRepository.existsById(id)){
+            wareCategoryRepository.deleteById(id);
+            return true;
+        }
+        return false;
```

Comment:
what do you mean false?
false - ware not exist?
false - could not delete the ware?
better change signature to void
and throw an exception if no ware present in db(ResourceNotFoundException)